app/components/search_bar/search_box.js

```diff
@@ -234,10 +244,24 @@ export default class Search extends Component {
                         duration: 200,
                     }
                 ).start(),
+                Animated.timing(
```

Comment:
we should: 
1. Use the nativeDriver for animations
2. Fire all the animations in parallel when possible